web/src/helpers/constants.js

```diff
@@ -35,11 +35,72 @@ const DEFAULT_FLAG_SIZE = 16;
 
 const DATA_FETCH_INTERVAL = 5 * 60 * 1000; // 5 minutes
 
+const LANGUAGE_NAMES = {
+  "ar": "اللغة العربية الفصحى",
+  "cs": "Čeština",
+  "da": "Dansk",
+  "de": "Deutsch",
+  "el": "Ελληνικά",
+  "en": "English",
+  "es": "Español",
+  "et": "Eesti",
+  "fi": "Suomi",
+  "fr": "Français",
+  "hr": "Hrvatski",
+  "id": "Bahasa Indonesia",
+  "it": "Italiano",
+  "ja": "日本語",
+  "ko": "한국어",
+  "nl": "Nederlands",
+  "no-nb": "Norsk (bokmål)",
+  "pl": "Polski",
+  "pt-br": "Português (Brasileiro)",
+  "ro": "Română",
+  "ru": "Русский язык",
+  "sk": "Slovenčina",
+  "sv": "Svenska",
+  "vi": "Tiếng Việt",
+  "zh-cn": "中文 (Mainland China)",
+  "zh-hk": "中文 (Hong Kong)",
+  "zh-tw": "中文 (Taiwan)"
+}
```

Comment:
Shouldn't these language codes match the casing used in #3948?
Same with the Facebook codes.

Reply:
Good point, I am actually a bit uncertain about the necessity of these files. It seems like we have a lot of dependencies on different language constants, eg this one, the FB ones, and the one in web/locales-config. Perhaps this is necessary to also keep docker-compose

cc @madsnedergaard

Reply:
Thanks and good catch! Updated now, I'll address the need and duplication of these constants in a separate PR later on instead of making this one more complicated :)